gnucash/price-quotes.scm

```diff
@@ -447,6 +450,11 @@
       (show-error (N_ "You are missing some needed Perl libraries.
 Run 'gnc-fq-update' as root to install them.")))
 
+     ((memq 'need-alphavantage-key fq-results)
+      (set! keep-going? #f)
+      (show-error (N_ "ERROR: ALPHAVANTAGE_API_KEY must be set for currency \
++ quotes; see https://wiki.gnucash.org/wiki/Online_Quotes#Source_Alphavantage.2C_US")))
```

Comment:
1, Can you put the URL as parameter in the translatable string? In the current form changing the URL would break all translations.
2. I know you spent much time to update wiki:Online_Quotes. But I still think a reference to the locally installed docs would be the better choice. Imagine, you were living in the outback and had only a very expensive, unstable internet connection.

Reply:
>locally installed docs
That would be nice but the only reference is in [Help Tips](https://code.gnucash.org/docs/C/gnucash-help/fq-sources.html) in a form that isn't readily linkable.

Reply:
Hi @fellen, I think the Online Quotes wiki is better because it has more info and it is all on the one page. The Help and the Guide are out of date. To use the manuals as is, you would have to search (not available in the web interface AFAICT) and jump all over the place. If you lived in the outback and have an unstable internet connection, you wouldn't be able to download the latest pdf/mobi manuals and probably would be unable to use online quotes. I'm not interested in maintaining the same documentation in 2 places...

Reply:
@goodvibes2 That's pretty much completely wrong. I already supplied the link to the exact page in the Help, so no searching required and it's up-to-date. It would be better still if the page had an anchor for the Alphavantage entry, hence my comment. The Mac and Windows bundles come with the Help and Guide built-in and it's a one-package addition on those Linux distros that don't include it as part of the main package. Gtk has a built-in context help mechanism that already uses the Help, click any of the Help buttons scattered about the program to see it in action. 

Your not being interested in maintaining the documentation is not a valid argument for not using it.